docs/porting_guide_v3.rst

```diff
@@ -0,0 +1,115 @@
+.. _porting_guide_v3:
+
+Ansible Builder version 3 Porting Guide
+=======================================
+
+This section discusses the behavioral changes between ``ansible-builder`` 1.2 and ``ansible-builder`` 3.0.
+
+It is intended to assist in updating your execution environment configuration files so they will work with this version of Ansible Builder.
```

Comment:
This sounds as if ansible-builder 3.0 no longer supports the v1 and v2 config formats. Is that really the case? I thought (and I think CI runs showed before I upgraded them to use the v3 format) that the old formats are also supported.

Reply:
I removed this line.